docs/wiki/EL/el-specs.md

```diff
@@ -0,0 +1,65 @@
+> [!NOTE]
+> This topic is open for collaboration on a separate branch
+
```

Comment:
The first header should be name of the topic with a quick intro, e.g.
# Execution Layer Specification

The EL is specified in multiple ways, in yellowpaper and executable python...